src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationRefresherProvider.cs

```diff
@@ -26,20 +41,21 @@ public AzureAppConfigurationRefresherProvider(IConfiguration configuration, ILog
                         // Use _loggerFactory only if LoggerFactory hasn't been set in AzureAppConfigurationOptions
                         if (refresher.LoggerFactory == null)
                         {
-                            refresher.LoggerFactory = _loggerFactory;
+                            refresher.LoggerFactory = loggerFactory;
```

Comment:
We dont need to check for null anymore. This comment is not applicable because there is no other way to set LoggerFactory now.